### ISSUE TITLE
Android: native Files app does not react on tap

### ISSUE THREAD

**Issue by ehabibov** [koush/vysor.io#1187]
**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://ps.uci.edu/~franklin/doc/file_upload.html (or any other file upload resource/app) and click "Upload file"
2. Choose native Files app and try to pick any media
3. Notice that file highlighted but nothing happens

**Expected behavior**
File being picked on tap/click.

**Screenshots**
![2022-12-13 21 43 07](https://user-images.githubusercontent.com/20490584/207432871-1a91c36f-bfeb-4c5b-bf2d-19b7d6b8bfed.gif)

**Desktop (please complete the following information):**
 - OS: MacOS Monterey 12.5.1

**Android (please complete the following information):**
 - Device: Samsung Galaxy A13
 - OS: Android 12
 - Vysor Bug Report Link: _Bug report is broken. It returns 'undefined' from clipboard._

**Additional context**
I notice it only for native Files app. I have another My Files app and Vysor taps work fine. Does not matter if that is upload from website or upload from any app - result is the same, no reaction for native Files.

**Comment by dieskim** [comment]
Yes I have noticed this on Samsung Devices as well...one trick is to use arrow keys and return to navigate, or just get a different file browser and use that.
